rootfs/api/models/app.py

```diff
@@ -375,14 +376,19 @@ def _destroy_containers(self, to_destroy):
         """Destroys containers via the scheduler"""
         if not to_destroy:
             return
-        destroy_threads = [Thread(target=c.destroy) for c in to_destroy]
-        [t.start() for t in destroy_threads]
-        [t.join() for t in destroy_threads]
-        [c.delete() for c in to_destroy if c.state == 'destroyed']
-        if any(c.state != 'destroyed' for c in to_destroy):
-            err = 'aborting, failed to destroy some containers'
-            log_event(self, err, logging.ERROR)
-            raise RuntimeError(err)
+
+        # for mock scheduler
+        if "scale" not in dir(self._scheduler):
```

Comment:
Had to do this for the mock scheduler to get rid of the last error in the unit tests that @bacongobbler noticed